code/modules/mob/living/carbon/human/human.dm

```diff
@@ -147,20 +147,28 @@
 		ear_damage += severity * 0.15
 		AdjustEarDeafness(severity * 0.5)
 
-	var/knockdown_value = min( round( severity*0.1  ,1) ,10)
-	if(knockdown_value > 0)
-		var/obj/item/Item1 = get_active_hand()
-		var/obj/item/Item2 = get_inactive_hand()
-		apply_effect(knockdown_value, WEAKEN)
-		var/knockout_value = min( round( damage*0.1  ,1) ,10)
-		apply_effect( knockout_value , PARALYZE)
-		apply_effect( knockout_value*2 , DAZE)
-		explosion_throw(severity, direction)
-
-		if(Item1 && isturf(Item1.loc))
-			Item1.explosion_throw(severity, direction)
-		if(Item2 && isturf(Item2.loc))
-			Item2.explosion_throw(severity, direction)
+	 /// Reduces effects by armor value.
+	var/bomb_armor_mult = ((CLOTHING_ARMOR_HARDCORE - bomb_armor) * 0.01)
+
+	if(severity >= 30)
+		flash_eyes(flash_timer = 4 SECONDS * bomb_armor_mult)
+
+	// Stuns are multiplied by 1 reduced by their medium armor value. So a medium of 30 would mean a 30% reduction.
+	var/knockdown_value = severity * 0.1
+	var/knockdown_minus_armor = min(knockdown_value * bomb_armor_mult, 1 SECONDS)
+	var/obj/item/Item1 = get_active_hand()
+	var/obj/item/Item2 = get_inactive_hand()
```

Comment:
we do not use capital letters in our var names carlarc

Reply:
i didnt make these vars they wer alredy heree!!!!!